shared/actions/tracker.js

```diff
@@ -149,6 +148,59 @@ export function onRefollow (username: string): TrackerActionCreator {
   }
 }
 
+export function onRefollow2 (username: string): TrackerActionCreator {
+  return (dispatch, getState) => {
+    const {trackToken} = (getState().tracker.trackers[username] || {})
+    const dispatchRefollowAction = () => {
+      dispatch({
+        type: Constants.onRefollow2,
+        payload: {username}
+      })
+    }
+    const dispatchErrorAction = () => {
+      dispatch({
+        type: Constants.onError,
+        payload: {username}
+      })
+    }
+
+    trackUser(trackToken)
+      .then(dispatchRefollowAction)
+      .catch(err => {
+        console.error("Couldn't track user:", err)
+        dispatchErrorAction()
```

Comment:
Now that we have a two-step action in the tracker popup, we can actually display errors there instead of just sending them to console.